### ISSUE TITLE
How we show value in fiat is problematic

### ISSUE THREAD

**Issue by ramijames** [GetScatter/ScatterDesktop#471]
So I have been staring at this the past hour or so and trying to figure out what bothers me so much about how we show value in fiat in Scatter.

![Screen Shot 2019-12-08 at 10 55 45 AM](https://user-images.githubusercontent.com/814302/70387090-6ef5ee80-19a9-11ea-8690-72e98785d629.png)

So at the top it is showing that I have $329.63 in my accounts total. 

In both my EOS accounts, I see money that I can't actually use because it is staked. I think we should rethink how we show purchased resources on the network because right now this is confusing.

A few concrete ideas:

1. Staked funds should not show up as value
2. We can show "Available funds" so that it is more clear that this fiat value is fungible now.
3. I'd like to make it more clear within this panel what the user is buying. The fact that CPU/NET is combined, but not really, also bothers the hell out of me.

![Screen Shot 2019-12-08 at 11 53 42 AM](https://user-images.githubusercontent.com/814302/70387752-a8325c80-19b1-11ea-91cd-095ef3925380.png)

**Comment by Soleone** [comment]
> 1. Staked funds should not show up as value

My two cents regarding showing staked tokens:

I'd argue users generally feel more worried when the amount shown is _less_ than they own (liquid or not). The priority is that your assets are still yours and nothing was lost somewhere.

Sure, it can be problematic if the amount shown is _higher_ than what is available right now because you thought you could cash out that much immediately, but I'd argue most users care more about _owning_ assets than _liquidating_ them.

For a lot of coins like EFX, I just stake all of them and care about the profits or resources it generates. The beauty of staking systems is that you keep all your value that you "invest" by locking something up, while also generating some new value for yourself. But I regularly come back to the wallet or bloks.io and check how much value in total I own right now, like if the number has been trending up or not.

TL;DR: In my opinion prioritize showing the sum of all owned tokens, no matter if immediately available or not.

> 2. We can show "Available funds" so that it is more clear that this fiat value is fungible now.

Yes, if changing anything rather show both, and make it clear what is available vs what is staked/locked. 👍

**Comment by ramijames** [comment]
In general I think that this entire mechanism is confusing and problematic. If the user has enough staked resources to perform actions, we should say everything is ok with their account. if not, they should be prompted to purchase more resources, with an estimate based on aggregate resource costs across the network. Basically, we should guess what a user needs and make them pre-buy and stake accordingly.
